.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [
+  push,
+  pull_request
+]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - id: vars
+        run: |
+          echo ::set-output name=go_version::$(curl -s https://raw.githubusercontent.com/actions/go-versions/main/versions-manifest.json | grep -oE '"version": "[0-9]{1}.[0-9]{1,}(.[0-9]{1,})?"' | head -1 | cut -d':' -f2 | sed 's/ //g; s/"//g')
```

Comment:
this seems overly complicated. If there's no easy way of doing matrices for Go versions then I'd say let's just use the latest version only. This project isn't really a Go lib that people with multiple versions rely on, but mostly a project that people will consume as binary or most likely as Docker container.

Reply:
I originally had this leftover code from other actions due to the manifest not always supplying the latest version, but it appears that's been resolved.